kaggle_cli/submit.py

```diff
@@ -129,29 +130,38 @@ def take_action(self, parsed_args):
             os.remove(target_name)
 
     @staticmethod
```

Comment:
Static method is not the best choice in such case. And an underscore prefix method indicates this method is private, which means it'll access the object's private fields, but it's not the case here. I suggest you extract the method to the module scope, i.e. move the `_make_archive_name` outside the `Submit` class but still in the `submit.py`.